src/components/FlowRunTimeline.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="flow-run-timeline-container">
+    <span class="flow-run-timeline-container__beta-badge">
+      Beta
+    </span>
```

Comment:
@pleek91 or @marichka-offen I thought I saw one of you create a beta badge somewhat recently; was that a one-off? If not we should (in another PR) componentize this so we're not repeating this markup.

Reply:
I created on in nebula. That whole component should probably get copied to orion-design and then removed from nebula.

Reply:
@brandonreid can you copy the badge from nebula into orion-design and use that here? Just so we can start using it consistently.

Reply:
Of course, my bad missen that